internal/scaffold/ansible/go_mod.go

```diff
@@ -43,36 +43,41 @@ go 1.13
 
 require (
 	github.com/operator-framework/operator-sdk master
-	sigs.k8s.io/controller-runtime v0.4.0
+	github.com/spf13/pflag v1.0.5
+	k8s.io/api v0.17.4
+	k8s.io/apimachinery v0.17.4
+	k8s.io/client-go v12.0.0+incompatible
```

Comment:
Do we need to actually scaffold these or can we continue relying on the go tooling to pull the correct ones in (like we have been)?